python/pyspark/ml/fpm.py

```diff
@@ -408,19 +429,26 @@ def getSequenceCol(self):
         """
         return self.getOrDefault(self.sequenceCol)
 
-    @since("2.4.0")
     def findFrequentSequentialPatterns(self, dataset):
         """
         Finds the complete set of frequent sequential patterns in the input sequences of itemsets.
 
-        :param dataset: A dataframe containing a sequence column which is
-                        `ArrayType(ArrayType(T))` type, T is the item type for the input dataset.
-        :return: A `DataFrame` that contains columns of sequence and corresponding frequency.
-                 The schema of it will be:
-                 - `sequence: ArrayType(ArrayType(T))` (T is the item type)
-                 - `freq: Long`
-
         .. versionadded:: 2.4.0
+
+        Parameters
+        ----------
+        dataset : :py:class:`pyspark.sql.DataFrame`
+            A dataframe containing a sequence column which is
+            `ArrayType(ArrayType(T))` type, T is the item type for the input dataset.
+
+        Returns
+        -------
+        pyspark.sql.dataframe.DataFrame
```

Comment:
nit: `pyspark.sql.DataFrame`?

Reply:
Thanks @huaxingao. I'll wait to see if you have any other comments and open a follow up.

Reply:
Ohh @huaxingao sorry I thought you finished the review. Sure, feel free to go ahead for a followup.

Reply:
I finished the review. please go ahead with your follow-up. I should have submitted all the comments at the same time instead of doing single comment.